src/cache/cache_las.c

```diff
@@ -761,6 +795,11 @@ __wt_las_insert_block(WT_CURSOR *cursor,
 
 	WT_ERR(__wt_block_manager_named_size(session, WT_LAS_FILE, &las_size));
 	WT_STAT_CONN_SET(session, cache_lookaside_ondisk, las_size);
+	max_las_size = ((WT_CURSOR_BTREE *)cursor)->btree->file_max;
+	if (max_las_size != 0 && (uint32_t)las_size > max_las_size)
```

Comment:
You should cast to `uint64_t` since that is what `max_las_size` is.

Reply:
ca144d1